swap_meet/item.py

```diff
@@ -1,2 +1,37 @@
+import uuid
 class Item:
-    pass
\ No newline at end of file
+    def __init__(self, id=None, condition=0):
+        self.condition = condition
+        self.id = id if id is not None else uuid.uuid4().int
```

Comment:
Nice work, using `is not None` ensures that you aren't disallowing `0` as an `id` for an item`.